viper-sys/build.rs

```diff
@@ -200,11 +200,30 @@ fn main() {
             java_class!("viper.silver.ast.Bool$", vec![
                 object_getter!(),
             ]),
-
+            
+            java_class!("viper.silver.ast.utility.FloatFactory", vec![
+                constructor!(),
+            ]),
+            java_class!("viper.silver.ast.utility.FloatFactory$", vec![
+                object_getter!(),
+                method!("apply", "(Lscala/Int;Lscala/Int;Lviper/silver/ast/utility/RoundingMode;)Lviper/silver/ast/utility/FloatFactory"),
```

Comment:
@vakaras I did not manage to match the type signature. I think the issue is `Lscala/Int`. I've also tried `Lscala/lang/Int` and `Lscala/lang/Int`. The class it refers to is [here](https://github.com/viperproject/silver/blob/8a7d18d1c9a418ea4fd644f9bed4410572f321dd/src/main/scala/viper/silver/ast/utility/BackendTypeFactories.scala#L45).

Reply:
You can look up the signatures with `javap`:
```bash
javap -cp viper_tools/backends/silicon.jar -s 'viper.silver.ast.utility.FloatFactory'
```

In this case, it is `(IILscala/Enumeration$Value;)Lviper/silver/ast/utility/FloatFactory;`